### ISSUE TITLE
LastModified is always NULL

### ISSUE THREAD

**Issue by dere-app** [SunnyApp/flutter_contact#23]
Hello,

The value of `lastModified` is always NULL, even after updating the contact.

Thanks

**Comment by ericmartineau** [comment]
@dere-app Which platform are you testing on?

**Comment by dere-app** [comment]
Hi @ericmartineau,

I'm using Android 10 (API 29), running the application on a Samsung S9+. Please see the output of `flutter doctor` below.

```
[✓] Flutter (Channel stable, 1.20.4, on Mac OS X 10.15.6 19G2021, locale en-GB)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 11.7)
[✓] Android Studio (version 3.6)
[✓] VS Code (version 1.49.0)
[✓] Connected device (1 available)

• No issues found!
```

**Comment by ericmartineau** [comment]
We recently removed the use of java8 `java.time.*` dates, in favor of the older `java.util.Date`.  This broke in a location where we used `OffsetDateTime.toString()`, which produces an ISO compatible date, where as `java.util.Date.toString()` does not.

Fixed in `0.6.4`
